<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountWindow</class>
 <widget class="QDialog" name="AccountWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>アカウント設定</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="userdata">
     <property name="title">
      <string>ユーザーデータ</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mail</string>
        </property>
        <property name="buddy">
         <cstring>userdata_mail</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="userdata_mail"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Pass</string>
        </property>
        <property name="buddy">
         <cstring>userdata_pass</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="userdata_pass"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ログインで使わなくても次枠追従などで必要となります</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="loginWay">
     <property name="title">
      <string>ログイン方法</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <widget class="QWidget" name="usersession_groupe" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="usersession_label">
           <property name="text">
            <string>ユーザーセッション</string>
           </property>
           <property name="buddy">
            <cstring>usersession</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="usersession">
           <property name="placeholderText">
            <string>user_session_12345678_abc...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QWidget" name="cookie_group" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="1">
          <widget class="QLineEdit" name="cookiesetting_filename"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="cookiesetting_file_label">
           <property name="text">
            <string>cookie db</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="cookiesetting_file_open_button">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="login_way_combo">
        <item>
         <property name="text">
          <string>firefox</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>直接入力</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>コメビュからログイン</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="get_session">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>↓取得</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ユーザーセッションはいずれかの方法で必ず入力してください</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QTextBrowser" name="userSessionWayDiscription"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AccountWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AccountWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
